### PR TITLE
Update worker-monitoring.adoc

### PR DESCRIPTION
**Pull request by MichelleZMY**
Update How Monitoring Works

### CODE REVIEW

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 1]
```diff
@@ -32,7 +32,7 @@ If you are setting this from the *Deploy Application* menu, click *Deploy Applic
 
 If you are setting this from the *Manage Application* menu, click *Apply Changes*.
 
-Upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
+For applications running on runtime 3.x.y, upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc]: For applications running on Mule 3.x.x...

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 2]
```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
 
 == How Monitoring Works
 
+*Runtime 3.x.y*
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc]: Mule runtime engine 3.x.x

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 3]
```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
 
 == How Monitoring Works
 
+*Runtime 3.x.y* 
+
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc:46]: Every 30 seconds, CloudHub sends an HTTP GET request...

Logs should be lowercase--logs

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 4]
```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
...
 Every thirty seconds, CloudHub sends an HTTP get request to the ping flow within the application. If the application is running normally, the ping flow sends a response. If CloudHub receives no response from the ping three times in a row, the Logs print a warning and display a full thread dump.
 
 image::ch-monitoring-threaddump.png[CH-monitoring-threaddump]
 
-In addition, CloudHub triggers a notification to the console to alert you that a worker for this application stopped responding.
+*Runtime 4.x.y*
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc]: Mule runtime engine 4.x.x

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 5]
```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
...
+
+Every thirty seconds, CloudHub checks the PID of the application. If CloudHub detects the application process is not running three times in a row, the Logs print a warning.
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc]: Logs should be lowercase--logs

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 6]
```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
...
+
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc:54]: I'm not sure what "any" application means.  This seems to be talking about a particular application (ie CloudHub checks the PID of "the" application). If we are talking about a particular app, then it should be as it was before..."this" application. If any means something else like a worker for all applications stopped responding, then let's use a different word.

**Review by valkyrie69 — CHANGES_REQUESTED** [review]
Thank you for this update. Please update according to my comments.

File: modules/ROOT/pages/worker-monitoring.adoc (hunk 1 above)
Comment (nishantkakar) [reply modules/ROOT/pages/worker-monitoring.adoc]:
```suggestion
For applications running on runtime 3.8.x and below, upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
```

File: modules/ROOT/pages/worker-monitoring.adoc (hunk 2 above)
Comment (nishantkakar) [reply modules/ROOT/pages/worker-monitoring.adoc]:
```suggestion
*Mule Runtime 3.8.x and below* 
```

File: modules/ROOT/pages/worker-monitoring.adoc (hunk 4 above)
Comment (nishantkakar) [reply modules/ROOT/pages/worker-monitoring.adoc]:
```suggestion
*Mule Runtime 3.9.x and above*
```

File: modules/ROOT/pages/worker-monitoring.adoc (hunk 5 above)
Comment (nishantkakar) [inline modules/ROOT/pages/worker-monitoring.adoc]: Should we write PID of the Mule Process instead?

**Review by nishantkakar — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/worker-monitoring.adoc (hunk 1 above)
Comment (valkyrie69) [reply modules/ROOT/pages/worker-monitoring.adoc]: Don't use "below." 3.8.x and earlier (less ambiguous)

File: modules/ROOT/pages/worker-monitoring.adoc (hunk 6 above)
Comment (MichelleZMY) [reply modules/ROOT/pages/worker-monitoring.adoc:54]: We are talking about a particular app but trying to say that "CloudHub triggers a notification to the console when application stopped responding" applies to both 3.x.x and 4.x.x apps, no matter if they are checking ping flow or PID

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 7]
```diff
@@ -32,7 +32,7 @@ If you are setting this from the *Deploy Application* menu, click *Deploy Applic
 
 If you are setting this from the *Manage Application* menu, click *Apply Changes*.
 
-Upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
+For applications running on runtime 3.8.x and below, upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc]: 3.8.x and earlier (below is ambiguous)

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 8]
```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
 
 == How Monitoring Works
 
-Every thirty seconds, CloudHub sends an HTTP get request to the ping flow within the application. If the application is running normally, the ping flow sends a response. If CloudHub receives no response from the ping three times in a row, the Logs print a warning and display a full thread dump.
+*Mule Runtime 3.8.x and below* 
+
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc:46]: 3.8.x and earlier

**Review by valkyrie69 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/worker-monitoring.adoc [hunk 9]
```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
...
+Every 30 seconds, CloudHub sends an HTTP get request to the ping flow within the application. If the application is running normally, the ping flow sends a response. If CloudHub receives no response from the ping three times in a row, the logs print a warning and display a full thread dump.
 
 image::ch-monitoring-threaddump.png[CH-monitoring-threaddump]
 
-In addition, CloudHub triggers a notification to the console to alert you that a worker for this application stopped responding.
+*Mule Runtime 3.9.x and above*
```
Comment (valkyrie69) [inline modules/ROOT/pages/worker-monitoring.adoc]: 3.9.x and later

File: modules/ROOT/pages/worker-monitoring.adoc (hunk 6 above)
Comment (valkyrie69) [reply modules/ROOT/pages/worker-monitoring.adoc:54]: Can we say something like:
For Mule 3.x.x and 4.x.x apps, CloudHub triggers a notification...
?

Comment (MichelleZMY) [reply modules/ROOT/pages/worker-monitoring.adoc:54]: Or can we say: In both cases, CloudHub triggers a notification... ?

Comment (valkyrie69) [reply modules/ROOT/pages/worker-monitoring.adoc:54]: @MichelleZMY That sounds good I think.

**Review by valkyrie69 — APPROVED** [review]

**Review by nishantkakar — APPROVED** [review]

**Pull request merged**